auxiliary_files/mumble-server.ini

```diff
@@ -185,7 +185,8 @@ allowping=true
 
 ; When a user connects to a server they've already been on, by default the
 ; server will remember the last channel they were in and move them to it
-; automatically. Toggling this setting to false will disable that feature.
+; automatically. This feature will only work for registered users.
```

Comment:
I don't like this being mentioned only for this option. This is universally true for all per-user properties. We could mention it somewhere else where it is clear that this is the general behavior. If we start mentioning it for specific options, we'll have to add the same note to all such options.